fetch-utils.js

```diff
@@ -18,13 +18,25 @@ The games are stored in the database using this data model:
 */
 
 export async function createGame(game){
+
     // create a single new game in the games table using the above object
+    const response = await client
+        .from('games')
+        .insert([{
+            name1: game.name1,
```

Comment:
seems like `.insert([game])` would also work, since the keys are all the same

Reply:
I think what happened here is , I was troubleshooting a bug and ruled this block out by being explicit. Forgot to change it back after.